config/brakeman.ignore

```diff
@@ -120,26 +120,6 @@
       "confidence": "Medium",
       "note": ""
     },
-    {
-      "warning_type": "File Access",
-      "warning_code": 16,
-      "fingerprint": "7a37fb95c5cd8c5d93b1ac95817aa23d44b12dd4c5152a175c3a840cc8b2ca27",
-      "check_name": "SendFile",
-      "message": "Parameter value used in file name",
-      "file": "app/controllers/case_attachments_controller.rb",
-      "line": 11,
-      "link": "http://brakemanscanner.org/docs/warning_types/file_access/",
-      "code": "send_file(download_to_tmpfile(Case::Base.find(params[:case_id]).attachments.find(params[:id]).preview_key), :type => File.mime_type?(Case.find(params[:case_id]).attachments.find(params[:id]).preview_key), :disposition => \"inline\")",
-      "render_path": null,
-      "location": {
-        "type": "method",
-        "class": "CaseAttachmentsController",
-        "method": "show"
-      },
-      "user_input": "params[:id]",
-      "confidence": "Weak",
-      "note": ""
-    },
```

Comment:
@willmcb, just curious about this one not, this removal is due to the removal of the flu gem or  this warning is not relevant anymore more?

Reply:
I think it is not relevant any more as this code doesn't exist in the app now. But I'll re-run brakeman and make sure 👍